js/renovation/pager/pager-props.tsx

```diff
@@ -15,7 +15,8 @@ export default class PagerProps {
 
   @OneWay() pagesCountText?: string = messageLocalization.getFormatter('dxPager-pagesCountText')();
 
-  // visible: true,
```

Comment:
Why is comment not removed?

Reply:
Without comment, it's not clear how visible related to test code

Reply:
What do you mean? Maybe you mistake?